.buildkite/pipeline.yml

```diff
@@ -1,16 +1,16 @@
 # Nodes with values to reuse in the pipeline.
 common_params:
   plugins: &common_plugins
-  - &bash_cache automattic/bash-cache#v1.3.2: ~
+  - &bash_cache automattic/bash-cache#2.0.0: ~
   env: &common_env
-    IMAGE_ID: xcode-12.5.1
+    IMAGE_ID: xcode-13
```

Comment:
Could be worth waiting for the `xcode-13.1` image since it's supposed to be released soon by Jeremy (like, tomorrow)? 🤷

Reply:
Ok after seeing the CI fail with `xcode-13` image because of `bundler` version, I remembered that we'd need to also apply the temporary fixits for `gem install bundler` (like [here](https://github.com/wordpress-mobile/WordPress-iOS/blob/35819a3b8b150e7a8dad76a474950cace792cc17/.buildkite/commands/build-for-testing.sh#L3-L5)) if we were to use that image, while this will be fixed in the soon-coming `xcode-13.1` image, so better wait for that new image anyway and do that bump only once it's available.